### ISSUE TITLE
Wrong port value in LinuxGSM alerts

### ISSUE THREAD

**Issue by shalak** [vinanrra/Docker-Satisfactory#13]
The port value shown in Discord alerts should be `queryport`, not `port`.

I'm not sure how to best approach this. Should the Dockefile replace the content of `https://github.com/GameServerManagers/LinuxGSM/blob/master/lgsm/functions/alert_discord.sh`? Or maybe it's better to fix it over there? Is the `queryport` even visible in the scope of that script?

**Comment by vinanrra** [comment]
Why should be swap to `queryport`, it's giving you any error? And if yes, you should create a issue at [LinuxGSM](https://github.com/GameServerManagers/LinuxGSM/issues) repo, because they are the ones that maintain the script

**Comment by shalak** [comment]
When connecting to server in the game client, it expects the `queryport` - or am I mistaken?

**Comment by vinanrra** [comment]
> 

True, which port it's showing you in the alerts, could you send me a screnshoot of your Discord message?

**Comment by vinanrra** [comment]
> When connecting to server in the game client, it expects the `queryport` - or am I mistaken?

True, could you send me a screenshot of your Discord alert message?

**Comment by shalak** [comment]
Here you go:
<img width="614" alt="image" src="https://user-images.githubusercontent.com/1536554/151230275-eac30a4d-f811-442b-a5ad-1a6a79c8f02c.png">
